index.html

```diff
@@ -518,16 +523,16 @@
 this document doesn't address how to balance the different ethical web principles if they come into conflict.
 
 Privacy on the Web is primarily regulated by two forces: the architectural capabilities that the Web
-platform exposes (or does not), and laws in the various jurisdictions where the Web is used
-([[New-Chicago-School]]). These regulatory mechanisms are separate; a law in one country does not
+platform exposes (or does not expose), and laws in the various jurisdictions where the Web is used
+([[New-Chicago-School]], [[Standard-Bodies-Regulators]]). These regulatory mechanisms are separate; a law in one country does not
 (and should not) change the architecture of the whole Web, and likewise Web specifications cannot
 override any given law (although they can affect how easy it is to create and enforce law). The Web
 is not merely an implementation of a particular legal privacy regime; it has distinct features and
 guarantees driven by shared values that often exceed legal requirements for privacy.
 
 However, the overall goal of privacy on the Web is served best when technology and law complement
 each other. This document seeks to establish shared concepts as an aid to technical efforts to
-regulate privacy on the web. It may also be useful in pursuing alignment with and between legal
+regulate privacy on the Web. It may also be useful in pursuing alignment with and between legal
```

Comment:
We're evenly split between "the web" and "the Web". Does the TAG have a style guide somewhere that says which to use? I'd like to actually unify on one capitalization instead of tweaking individual cases.

Reply:
Concur

Reply:
```suggestion
regulate privacy on the web. It may also be useful in pursuing alignment with and between legal
```